cvat/apps/engine/views.py

```diff
@@ -977,6 +988,182 @@ def self(self, request):
         serializer = serializer_class(request.user, context={ "request": request })
         return Response(serializer.data)
 
+@method_decorator(name='list', decorator=swagger_auto_schema(
+        operation_summary='Returns a paginated list of storages according to query parameters',
+        manual_parameters=[
+                openapi.Parameter('provider_type', openapi.IN_QUERY, description="A supported provider of cloud storages",
```

Comment:
@Marishka17 , should we add "display name" here?

Reply:
Fixed